bot/bot.py

```diff
@@ -0,0 +1,54 @@
+import discord
+import responses
+
+
+async def send_message(message, user_message, is_private):
+    try:
+        response = responses.get_response(user_message)
+        await message.author.send(
+            response
+        ) if is_private else await message.channel.send(response)
+
+    except Exception as e:
+        print(e)
+
+
+def run_discord_bot():
+    TOKEN = ""
+    # please input your own token here
+    intents = discord.Intents.default()
+    intents.message_content = True
+    client = discord.Client(intents=intents)
+
+    @client.event
+    async def on_ready():
+        print(f"{client.user} is online")
+
+    @client.event
+    async def on_message(message):
+        if message.author == client.user:
+            return
+
+        username = str(message.author)
+        user_message = str(message.content)
+        channel = str(message.channel)
+
+        print(f'{username} said: "{user_message}" {{channel}}')
+
+        if user_message[0] == "?":
+            user_message = user_message[1:]
+            await send_message(message, user_message, is_private=True)
+        else:
+            await send_message(message, user_message, is_private=False)

```

Comment:
will this cover the event for a user posting a new resume to the channel?

I dont know if we'll be reading messages themselves, but more of the actual post, and then reading its associated tags right?

Reply:
I will have to reread Discord API docs but if I recall correctly, you are right. I think we have to read for the creation of a new Thread(which can contain multiple messages) and its associated tags